x/protorev/keeper/rebalance_test.go

```diff
@@ -737,6 +737,7 @@ func (s *KeeperTestSuite) TestRemainingPoolPointsForTx() {
 }
 
 func (s *KeeperTestSuite) TestUpdateSearchRangeIfNeeded() {
+	s.SetupTest()
```

Comment:
This is almost certainly a bug, shouldn;t be needed.
```suggestion
```

Reply:
It was needed, when default was non zero. Now that it is zero, this can be removed

Reply:
https://github.com/osmosis-labs/osmosis/pull/6034/commits/708442ed47c77ebc6bfdf6178172c48e6fda94e2